FlowForms-Core/src/commonMain/kotlin/com/rootstrap/flowforms/core/field/DefaultFieldValidationBehavior.kt

```diff
@@ -107,15 +83,13 @@ open class FField(
         }
 
         val fieldStatus = when {
-            failedValResults.isEmpty() -> FieldStatus(CORRECT, validationResults)
+            failedValResults.isEmpty() -> FieldStatus(StatusCodes.CORRECT, validationResults)
             failedValResults.size == 1 -> FieldStatus(failedValResults.first().resultId, validationResults)
-            else -> FieldStatus(INCORRECT, validationResults)
+            else -> FieldStatus(StatusCodes.INCORRECT, validationResults)
         }
 
-        _status.value = fieldStatus
-        return fieldStatus.code == CORRECT
+        mutableFieldStatus.value = fieldStatus
+        return fieldStatus.code == StatusCodes.CORRECT
     }
 
-    class ValidationShortCircuitException(val validationResult : ValidationResult) : Exception()
-
-}
+}
```

Comment:
add an extra line at the end of the file

Reply:
Done :)